modules/nf-core/chopper/main.nf

```diff
@@ -0,0 +1,36 @@
+process CHOPPER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::chopper=0.3.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/chopper:0.3.0--hd03093a_0':
+        'quay.io/biocontainers/chopper:0.3.0--hd03093a_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta)                 , emit: meta
+    path "versions.yml"             , emit: versions
+    tuple path("*.filtered.fastq.gz")  , emit: fastq
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    zcat $fastq | \\
+    chopper \\
+        --threads $task.cpus \\
+        $args | \\
+    gzip >  ${prefix}.filtered.fastq.gz
```

Comment:
```suggestion
    zcat \\
        args
        $fastq | \\
    chopper \\
        --threads $task.cpus \\
        $args2 | \\
    gzip \\
        args3 \\
        >  ${prefix}.fastq.gz
```

Need args for all steps :)

I've also removed `filtered` from the output as this needs to be up to the pipeline developer. To prevent a risk of filename clash, please see how to add a warning in teh docs [^1]. You will need to update the prefix in `nextflow.config` too to customiuse the name for your tests.

[^1]: https://nf-co.re/docs/contributing/modules#naming-conventions point 6

Reply:
should be fixed in new commit